Doc/library/os.rst

```diff
@@ -732,6 +764,45 @@ process and user.
       The function is now always available and is also available on Windows.
 
 
+.. function:: unshare(flags)
+
+   Disassociate parts of the process execution context, and move them into a
+   newly created namespace.
+   See the :manpage:`unshare(2)`
+   man page for more details.
+   The *flags* argument is a bit mask combining zero or more of the ``CLONE_*``
+   constants using ``|`` (bitwise or), that specifies which parts of the execution
+   context should be unshared from their existing associations and moved to a
+   new namespace..
+   If *flags* is specified as zero, no changes are made to the calling process's
```

Comment:
```suggestion
   If the *flags* argument is ``0``, no changes are made to the calling process's
```

Use an actual literal where appropriate, avoid awkwardness over flags being singular and eliminate extra wordiness